apps/test/unit/templates/rubrics/LearningGoalsTest.jsx

```diff
@@ -80,6 +90,35 @@ const studentLevelInfo = {
 };
 
 describe('LearningGoals - React Testing Library', () => {
+  let annotatorStub,
+    annotateLineStub,
+    highlightLineStub,
+    clearAnnotationsStub,
+    clearHighlightedLinesStub;
+
+  // Stub out our references to the singleton and editor
+  beforeEach(() => {
+    let annotatorInstanceStub = sinon.stub();
+    annotatorInstanceStub.getCode = sinon.stub().returns(code);
+    annotatorStub = sinon
+      .stub(EditorAnnotator, 'annotator')
+      .returns(annotatorInstanceStub);
+    annotateLineStub = sinon.stub(EditorAnnotator, 'annotateLine');
+    clearAnnotationsStub = sinon.stub(EditorAnnotator, 'clearAnnotations');
+    highlightLineStub = sinon.stub(EditorAnnotator, 'highlightLine');
+    clearHighlightedLinesStub = sinon.stub(
+      EditorAnnotator,
+      'clearHighlightedLines'
+    );
+  });
+  afterEach(() => {
+    annotatorStub.restore();
+    annotateLineStub.restore();
+    clearAnnotationsStub.restore();
+    highlightLineStub.restore();
+    clearHighlightedLinesStub.restore();
+  });
+
```

Comment:
nit: unless they are needed to keep other tests from breaking, I think these better belong within the `  describe('annotateLines', () => {` block currently starting on line 266

Reply:
I DRY'd them a little by putting those stubs/restores in functions so they could be then used in before/after in the `annotateLines` and `clearAnnotations` tests in that section.